build.sbt

```diff
@@ -20,7 +20,7 @@ import java.io.File
 // ============================================================================
 
 val scalacVersion = "2.13.11"
-// Since the release of GraalVM 23.0.0, the versioning is the same for Graal and OpenJDK.
+// GraalVM versions should be synchronized with the build-config.yaml
 val graalVersion = "17.0.7"
```

Comment:
Can we have a test/code that reads `build-config.yaml` and compares the values? `build.sbt` is fully Turing complete, so checking the consistency should be possible. Having an active check, rather than a warning will alert us in the future and help us keep consistency.

Reply:
I removed versions from the `build-config.yaml`. Now `run` script gets them directly from `build.sbt`.